### PR TITLE
DRA-1512: facet call split out from results and more.

### PR DESCRIPTION
**Pull request by jesperlauridsen**
- Facets are now called separately. If facets are opened when we do a new search, they're called, otherwise first when the "show filters" are opened. This gives a little loadtime, but it's fine.
- Back button now goes to the last search, wherever you might have gone, or back to the frontpage if there was no search.
- Videoplayer new respons to the 1002 error from Kaltura, throwing an error and telling the user they cant access this recording.
- Fixed a bug with filters not being applied right in the timeSearchStore.
- Fixed a few background accessibility issues.
- Removed a few console.logs

### CODE REVIEW

File: src/components/common/timeSearch/TimeSearchFilters.vue [hunk 1]
```diff
@@ -717,14 +717,14 @@ fieldset {
 	pointer-events: none;
 	z-index: 5;
 	left: -1px;
-	background: linear-gradient(-90deg, rgba(215, 255, 98, 0) 0%, rgb(250, 250, 250) 95%);
+	/* background: linear-gradient(-90deg, rgba(215, 255, 98, 0) 0%, rgb(250, 250, 250) 95%); */
```
Comment (Pbruun) [inline src/components/common/timeSearch/TimeSearchFilters.vue:720]: Should be removed?

File: src/components/common/timeSearch/TimeSearchFilters.vue [hunk 2]
```diff
@@ -717,14 +717,14 @@ fieldset {
...
 }
 
 .slider-whiteoff-container:after {
 	right: -1px;
 	left: initial;
 	top: 0px;
-	background: linear-gradient(90deg, rgba(215, 255, 98, 0) 0%, rgb(250, 250, 250) 95%);
+	/* background: linear-gradient(90deg, rgba(215, 255, 98, 0) 0%, rgb(250, 250, 250) 95%); */
```
Comment (Pbruun) [inline src/components/common/timeSearch/TimeSearchFilters.vue:727]: Should be removed?

File: src/views/ShowRecord.vue [hunk 3]
```diff
@@ -74,8 +76,34 @@ export default defineComponent({
...
 		const authStore = useAuthStore();
 		const route = useRoute();
+		const router = useRouter();
 		const loading = ref(true);
 		const contentNotAllowed = ref(false);
+		const backLink = ref('');
+
+		onMounted(async () => {
+			let back = router.options.history.state.back as string;
+			console.log(back);
```
Comment (Pbruun) [inline src/views/ShowRecord.vue:86]: console.log

**Review by Pbruun — COMMENTED** [review]
Looks great!

**Comment by Pbruun** [comment]
Merge conflict to be resolved

File: src/components/common/timeSearch/TimeSearchFilters.vue (hunk 1 above)
Comment (jesperlauridsen) [reply src/components/common/timeSearch/TimeSearchFilters.vue:720]: Im just outcommenting it here - we might wanna look at bringing this back at a later date - if we can figure out a way around the accessibility issues here.

File: src/components/common/timeSearch/TimeSearchFilters.vue (hunk 2 above)
Comment (jesperlauridsen) [reply src/components/common/timeSearch/TimeSearchFilters.vue:727]: Im just outcommenting it here - we might wanna look at bringing this back at a later date - if we can figure out a way around the accessibility issues here.

**Pull request merged**